src/app/page.tsx

```diff
@@ -1,7 +1,20 @@
-export default function Home() {
+'use client';
+
+import Form from '@/components/Form';
+import { useRouter } from 'next/navigation';
+import './globals.css';
+
+export default function RootLayout() {
+  const router = useRouter();
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    router.push('/result');
```

Comment:
[Q]
値の渡し方について、実装方針の相談になります。（もしこのPRのスコープ外でしたらすみません！説明かチケットに明記していただけますと🙏）


先々の話しになりますが、result画面にフォロワー数を渡して、result画面側でスコアを算出するイメージでいますか？それなら、ここでクエリパラメータとして入力してもらったフォロワー数を渡してもいいと思います。

例として、以下みたいな感じでやるとクエリパラメータに 100 が渡ります。
```suggestion
    router.push(`/result?followers=${100}`);
```


もしくは、この関数の中で作っていただいた関数を呼び、スコアを算出してからリザルト画面を呼ぶでもいい気がします〜

Reply:
ログイン側で、数値を取得してきてそれを計算ロジックで、戦闘力定義したものをresult画面に渡す方が良さそう。
resultでは、結果のみの実装とし、役割を分担した方が良さそう。

Reply:
getBattlePower されていることを確認